tests/queries/0_stateless/01277_alter_rename_column_constraint_expr.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE IF EXISTS table_for_rename;
```

Comment:
I have stolen this test in my PR https://github.com/ClickHouse/ClickHouse/pull/10847 :)